.github/workflows/publishbot.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Robot
+on:
+  push:
+    branches:
+      - master
+    tags: ['*']
```

Comment:
As opposed to `masterbot.yml`, I intend for this workflow to run on tagged commits.